sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>Confirming that this isn't supported by <code>sagemaker.image_uris.retrieve()</code> , so there's no alternative to hard-coding regions? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yes, this is not available using retrieve() and no other alternative.